docs/gitlab.md

```diff
@@ -21,6 +21,30 @@ To use on another instance, eg. `https://gitlab.example.com` requires setup and
     git config --global --unset-all credential.https://gitlab.example.com.provider
 ```
 
+### Config for popular instances
+
+For convenience, here are the config commands for several popular GitLab instances:
```

Comment:
It could also be a good idea to clarify that these instances are community-maintained and are not managed by the GCM project.